Tests/SwiftParserTest/ExpressionTests.swift

```diff
@@ -577,7 +577,8 @@ final class ExpressionTests: XCTestCase {
         )
       ],
       fixedSource: ##"""
-        """""""
+        """"
+        """
```

Comment:
In this case it should actually be

```swift
"""
"
"""
```

That's properly the parser not doing things 100% correctly.